testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -91,6 +95,22 @@ module 0xABCD::Simple {
         counter.count;
     }
 
+    public entry fun step_destination(owner: &signer, destination: address) acquires Counter {
```

Comment:
It'd be closer to Renfield and Metapixel minting if we add a batch index here (have multiple different counters that correspond to different batch indices) and have different transmitter instances use those different indices to simulate the account pool that Renfield and Metapixel have.

Reply:
not equivalent, but argument:
  num_modules: 1,
allows us to control parallelism. 

instead of having multiple resources in a single module, we publish the same module multiple times, each with single resource. I think that is fine for now.

considering the results (https://aptos-org.slack.com/archives/C03F71PU3B5/p1680760358119509), this is anyways more efficient than those contracts, so we will be adding token minting and other closer operations as well

Reply:
Using the same module would change the efficiency actually since caching would kick in

Reply:
caching is per block. If we have working set of 10 modules, that is going to have the same performance (given thousands of transactions in a block). 
If we set to 1000 modules, then yes, module loading will change things.

Reply:
And to clarify, goal of this method is not to be representative of Renfield/Metapixel, but to have a simplest serial workload that we can benchmark and compare to others in isolation

We will have separate transaction types for Renfield/Metapixel-like workload, and should make sure those are representative as much as possible